src/components/predictive-layer.vue

```diff
@@ -120,7 +116,7 @@
         </h1>
         <PopularSearches
           :animation="suggestionsAnimation"
-          :max-items-to-render="5"
+          :max-items-to-render="3"
```

Comment:
I think that the max popular searches to render are 4

```suggestion
          :max-items-to-render="4"
```